web/components/SchedulePage.jsx

```diff
@@ -1,33 +1,149 @@
+            this.props.setSelectedDay(today)
+        }
+    }
+
+    renderSetBlocks = (selectedDay) => {
+        const { currentTeamMember } = this.props;
+        selectedDay = moment(selectedDay).format('YYYY-MM-DD')
+        const setBlocksByDate = _.groupBy(currentTeamMember.weeklySetblocks, 'date')
+        const setBlocks = setBlocksByDate[selectedDay];
+        if (setBlocks) {
+            return setBlocks.map(setBlock => {
+                return <SetBlock data={setBlock} key={setBlock.id} />
+            })
+        } else {
             return (
-                <h6>
-                teamMemberId:
-                    {match.params.teamMemberId}
-                </h6>
+                <Text weight='600' aling='center'> This user hasn't committed any Setblocks for this day </Text>
             )
         }
+    }
+
+    getDaysOfWeek = () => {
+        const startOfWeek = moment()
+        .startOf('isoWeek');
+        const endOfWeek = moment()
+        .endOf('isoWeek');
+
+        const days = [];
+        let day = startOfWeek;
+
+        while (day <= endOfWeek) {
+            days.push(day.toDate());
+            day = day.clone()
+            .add(1, 'd');
+        }
+
+        this.setState({
+            daysOfWeek: days,
+        })
+        this.props.setSelectedDay(days[0])
+    }
 
-        if (match.params.dayOfWeek) {
+    renderIfItReady() {
+        const {
+            match, currentTeamMember, fetchingData, selectedDay 
+        } = this.props
+        if (fetchingData) {
+            return ( // If you are waiting for the API to respond, render a loading
+                <Flex
+                    center
+                    row
+                >
+                    <Text weight='900'>Loading</Text>
+                    <LoadingDots />
+                </Flex>
+            )
+        } else {
             return (
-                <h6>
-                    dayOfWeek:
-                    {match.params.dayOfWeek}
-                </h6>
+                <Flex
+                    center
+                    column
+                >
+                    <Text
+                        weight='900'
+                        aling='center'
+                        style={{ borderBottom: '1px solid red' }}
+                    >
+                        {match.params.teamMemberId ? currentTeamMember.name : 'Your'}
+                        {' Schedule\'s Page'}
+                    </Text>
+                    {this.renderSetBlocks(selectedDay)}
+                </Flex>
             )
         }
     }
 
     render() {
-        const { match } = this.props;
+        const { selectedDay } = this.props;
+        const { daysOfWeek } = this.state;
         return (
-            <div className='SchedulePage'>
-                <h6>SchedulePage</h6>
-                {this.renderBaseOnParams(match)}
-            </div>
+            <Flex
+                row
+                flexDirection='horizontal'
```

Comment:
I don't believe this syntax is necessary for Flex, the `row` prop should do the trick. Tracking this in #22 for future cleanup since I'd like to just merge this PR and leave these non-critical items for release prep